ConsoleApplication4/ConsoleApplication4/WheatPizzaBuilder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace ConsoleApplication4
+{
+    class WheatPizzaBuilder:IPizzaBuilder
+    {
+        private WheatPizza wheatepizza;
+        public WheatPizzaBuilder()
+        {
+            this.reset();
+        }
+        public void reset()
+        {
+            this.wheatepizza = new WheatPizza();
+
+        }
+        public void BuildDough()
+        {
+            wheatepizza.Add("Wheat Dough + ");
+        }
+
+        public void BuildSauce()
+        {
+            wheatepizza.Add("Sauce + ");
+        }
+
+        public void BuildTopping()
```

Comment:
It would be nice if you specified some toppings like (Broccoli, Mushroom, onions) or what you want rather than just writing the word Topping.